wezterm-ssh/src/sessioninner.rs

```diff
@@ -517,8 +525,16 @@ impl SessionInner {
 
             let stdin = &mut chan.descriptors[0];
             if stdin.fd.is_some() && !stdin.buf.is_empty() {
-                write_from_buf(&mut chan.channel.writer(), &mut stdin.buf)
```

Comment:
That's because I'm seeing weird error returns from here, causing wezterm to give up shortly after a ssh-agent forward sessions ended, even if there's still other channels (mainly, shell) connected.

I tried to write a e2e test for this, but unfortunately I can only reproduce on Windows against KeeAgent, and the testing infra seems to support Linux/Mac only.